src/loaders/swaggerLoader.ts

```diff
@@ -0,0 +1,29 @@
+import { MicroframeworkLoader, MicroframeworkSettings } from 'microframework';
+import * as swaggerUi from 'swagger-ui-express';
+// tslint:disable-next-line:no-var-requires
+const swaggerJSDoc = require('swagger-jsdoc');
+
+import { env } from '../env';
+
+export const swaggerLoader: MicroframeworkLoader = (settings: MicroframeworkSettings | undefined) => {
+    if (settings && env.swagger.enabled) {
+        const expressApp = settings.getData('express_app');
+        const swaggerOptions = {
+            swaggerDefinition: {
+                components: {},
+                basePath: `${env.app.routePrefix}`,
+                info: {
+                    title: env.app.name,
+                    description: env.app.description,
+                    servers: [{
+                        url: `${env.app.schema}://${env.app.host}:${env.app.port}${env.app.routePrefix}`,
+                    }],
+                },
+            },
+            explorer: true,
+            apis: ['**/*.ts'],
```

Comment:
Add only controllers over here

Reply:
@VivekFitkariwala Added only controllers path in the apis array.